data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Telephone(mobile or home):
```

Comment:
I think this is a bit creepy for coala to collect.

Google probably does collect it, and any time a ph. is needed, Google should probably be involved.